src/main/java/org/folio/bulkops/service/ErrorService.java

```diff
@@ -116,9 +113,11 @@ public void saveErrorsFromDataImport(UUID bulkOperationId, UUID dataImportJobId)
         String identifier = EMPTY;
         try {
           if (identifierType == IdentifierType.ID) {
-            identifier = srsClient.getSrsRecordById(errorEntry.getSourceRecordId()).getExternalIdsHolder().getInstanceId();
+            var idList = errorEntry.getRelatedInstanceInfo().getIdList();
```

Comment:
I would propose the following refactoring to avoid duplications:
          List<String> identifiersList = new ArrayList<>();
          var relatedInstanceInfo = errorEntry.getRelatedInstanceInfo();
          if (identifierType == IdentifierType.ID) {
            identifiersList = relatedInstanceInfo.getIdList();
          } else if (identifierType == IdentifierType.INSTANCE_HRID) {
            identifiersList = relatedInstanceInfo.getHridList();
          }
          identifier = identifiersList.isEmpty() ? null : identifiersList.get(0);

Reply:
Refactored it without new ArrayList